.gitignore

```diff
@@ -2,13 +2,8 @@
 .idea
 build/
 target/
-build/out
-service/out
-component-test/out
-api/out
-service/build
-component-test/build
-api/build
+**/out/
```

Comment:
@awasum the out directory has been ignored here. :)

Reply:
OK, thanks. I missed it.